### PR TITLE
Gun-Slingers.

### PR DESCRIPTION
**Pull request by ncr38**

### CODE REVIEW

**Comment by ncr38** [comment]
@palnabarun  @Shubhanshu07

File: README.md [hunk 1]
```diff
@@ -1 +1,48 @@
 # Gun-Slingers
+_A Fun Event_
+___
+## Abstract
+___
+The idea of this project was to make a game consisting of two small robots to play with each other.Both robots were loaded with 5 table tennis balls each . Both robots were placed on a table in their respective playing zones.Both were manually controlled via USB remote controller.Each player fire balls from their robot to the opponent's robot.The player with more hits on the opponent wins the game.
+![alt](model.jpg)
+___
+## Team Members
```
Comment (agshubhs) [inline README.md]: @ncr38 write the name in alphabetical order

File: README.md [hunk 2]
```diff
@@ -1 +1,48 @@
...
+1. Kartik Chimpa
+2. Satyansh Rai
+3. Prakhar Mishra
+4. Pratikesh Thakur
+5. Sourabh Patil
+6. Vibhore Verma
+7. Abhishek Mishra
+8. Navin Chandra Rai
+9. Vipul Goel
+___
```
Comment (agshubhs) [inline README.md]: @ncr38  Remove the horizontal line from the readme file

**Comment by agshubhs** [comment]
@ncr38 make the necessary changes. 
Rest is nice.

**Comment by palnabarun** [comment]
Looks good to me.

File: README.md (hunk 2 above)
Comment (agshubhs) [reply README.md]: Nice

**Review by agshubhs — COMMENTED** [review]
nice

**Pull request merged**